src/lib/wpack.js

```diff
@@ -57,22 +57,30 @@ const createConfigs = (fns, config, servicePath, defaultOutput, folder) =>
  * @param {array} configs Array of webpack configurations
  * @returns {Promise} Webpack stats
  */
-const run = configs =>
+const run = (configs, sls) =>
   new Promise((resolve, reject) => {
-    webpack(configs, (err, stats) => {
-      if (err) reject(`Webpack compilation error: ${err}`);
+    configs.map(config => () =>
+      new Promise((res, rej) => {
+        webpack(config, (err, stats) => {
+          sls.cli.log(`Creating: ${Object.keys(config.entry)[0]}`);
+          if (err) reject(`Webpack compilation error: ${err}`);
 
-      console.log(stats.toString({ // eslint-disable-line no-console
-        colors: true,
-        hash: false,
-        chunks: false,
-        version: false,
-      }));
+          console.log(stats.toString({ // eslint-disable-line no-console
+            colors: true,
+            hash: false,
+            chunks: false,
+            version: false,
+            cache: false,
```

Comment:
Ye, you're right. Just checked everything off. The fix is not returning the resulted stats from the promise. Otherwise it stacks it and it'll run into a node memory error if it becomes too big.